python/src/RandomVectorMetropolisHastings_doc.i.in

```diff
@@ -42,22 +42,22 @@ We can also link the parameters of the random vector to the state of the chain,
 here the parameters of the random vector being the ones of the underlying distribution:
 
 >>> slf = ot.SymbolicFunction(['x'], ['0.0', '0.1'])
```

Comment:
```suggestion
>>> linkFunction = ot.SymbolicFunction(['x'], ['x', '0.1'])
```
That way `sampler` becomes a random walk, which is a little more interesting than having the parameters be independent from the state of the Markov chain.